src/sdc11073/xml_types/pm_types.py

```diff
@@ -1253,176 +1216,185 @@ def __init__(self, patient=None, assigned_location=None, visit_number=None, dang
+        pm.CalibrationDocumentation, value_class=CalibrationDocumentation)
+    ComponentCalibrationState: CalibrationState | None = cp.EnumAttributeProperty('ComponentCalibrationState',
+                                                                                  enum_cls=CalibrationState)
+    Type: CalibrationType | None = cp.EnumAttributeProperty('Type',
+                                                            implied_py_value=CalibrationType.UNSPEC,
+                                                            enum_cls=CalibrationType)
+    Time: float | None = cp.TimestampAttributeProperty('Time')
     _props = ['CalibrationDocumentation', 'ComponentCalibrationState', 'Type', 'Time']
 
 
 class ApprovedJurisdictions(PropertyBasedPMType):
-    # pylint: disable=invalid-name
+    """Represents BICEPS ApprovedJurisdictions."""
+
     NODETYPE = pm.ApprovedJurisdictions
-    ApprovedJurisdiction = cp.SubElementListProperty(pm.ApprovedJurisdiction,
+    ApprovedJurisdiction: list[InstanceIdentifier] = cp.SubElementListProperty(pm.ApprovedJurisdiction,
                                                      value_class=InstanceIdentifier)
     _props = ['ApprovedJurisdiction']
 
 
 class ContainmentTreeEntry(PropertyBasedPMType):
-    Type = cp.SubElementProperty(pm.Type, value_class=CodedValue, is_optional=True)
-    _props = ['type']
+    """Represents BICEPS ContainmentTree -> Entry."""
+
+    Type: CodedValue | None = cp.SubElementProperty(pm.Type, value_class=CodedValue, is_optional=True)
+    _props = ['Type']
 
 
 class ContainmentTree(PropertyBasedPMType):
-    Entry = cp.SubElementListProperty(pm.Entry, value_class=ContainmentTreeEntry)
+    """Represents BICEPS ContainmentTree."""
+
+    Entry: ContainmentTreeEntry = cp.SubElementListProperty(pm.Entry, value_class=ContainmentTreeEntry)
     _props = ['Entry']
 
 
 # Technically, Retrievalibity belongs to msg_types, because it is defined in BICEPS message model.
 # But its usage is in mdib, not in messages, it fits here better.
 # This also helps to avoid a circular dependency.
 class RetrievabilityMethod(StringEnum):
+    """Represents BICEPS message model RetrievabilityMethod."""
+
     GET = 'Get'
     PERIODIC = 'Per'
     EPISODIC = 'Ep'
     STREAM = 'Strm'
 
 
 class RetrievabilityInfo(PropertyBasedPMType):
+    """Represents BICEPS message model RetrievabilityInfo."""
+
     NODETYPE = msg.RetrievabilityInfo
-    Method = cp.EnumAttributeProperty('Method', enum_cls=RetrievabilityMethod, is_optional=False)
-    UpdatePeriod = cp.DurationAttributeProperty('UpdatePeriod', implied_py_value=1.0)
+    Method: RetrievabilityMethod = cp.EnumAttributeProperty('Method',
+                                                            enum_cls=RetrievabilityMethod,
+                                                            is_optional=False)
+    UpdatePeriod: DurationType = cp.DurationAttributeProperty('UpdatePeriod', implied_py_value=1.0)
```

Comment:
Argument "implied_py_value" to "DurationAttributeProperty" has incompatible type "float"; expected "Optional[DurationType]"